lib/status-bar-item.js

```diff
@@ -34,3 +38,14 @@ function lineEndingName (lineEndings) {
     return ''
   }
 }
+
+function lineEndingDescription (lineEndings) {
+  const convert = {
+    Mixed: 'mixed',
+    LF: 'LF (Unix)',
+    CRLF: 'CRLF (Windows)',
+    CR: 'CR (old Macintosh)'
```

Comment:
can we changed "old Macintosh" to "Classic MacOS" ? That seems to be the most accurate way of describing Mac OS 1-9 given articles like https://en.wikipedia.org/wiki/Classic_Mac_OS